src/sqlacodegen/generators.py

```diff
@@ -13,7 +13,7 @@
 from keyword import iskeyword
 from pprint import pformat
 from textwrap import indent
-from typing import Any, ClassVar, Literal, cast
+from typing import Any, ClassVar, Literal, _SpecialForm, cast
```

Comment:
I'm not super fond of importing undocumented classes. For the special case of `Any`, can we just hard-code its name?

Reply:
See my suggestion below.

Reply:
Suggestion accepeted. Resolving